packages/components/Tabs/src/Tabs.tsx

```diff
@@ -82,10 +91,33 @@ export const Tabs = compose<TabsType>({
 
     const styleProps = useStyling(userProps, (override: string) => state[override] || userProps[override]);
 
+    const onKeyDown = (ev: IKeyboardEvent) => {
+      if (ev.nativeEvent.key === 'ArrowRight' || ev.nativeEvent.key === 'ArrowLeft') {
+        const length = state.context.enabledKeys.length;
+        const currTabItemIndex = state.context.enabledKeys.findIndex(x => x == state.context.selectedKey)
+        let newCurrTabItemIndex;
+        if (ev.nativeEvent.key === 'ArrowRight') {
+          if (!(!isCircularNavigation && currTabItemIndex + 1 == length)) {
+            newCurrTabItemIndex = (currTabItemIndex + 1) % length;
+            state.context.selectedKey = state.context.enabledKeys[newCurrTabItemIndex];
+            data.onKeySelect(state.context.selectedKey);
+          }
+        }
+        if (ev.nativeEvent.key === 'ArrowLeft') {
+          if (!(!isCircularNavigation && currTabItemIndex == 0)) {
+            newCurrTabItemIndex = (currTabItemIndex - 1 + length) % length;
+            state.context.selectedKey = state.context.enabledKeys[newCurrTabItemIndex];
+            data.onKeySelect(state.context.selectedKey);
+          }
+        }
+      }
+    };
```

Comment:
Am I to understand this correctly and too that we basically implemented re-implemented focusZone with JS here?

Reply:
Yup, with a combination of setting components to focusable and this function, the needed behavior of the FocusZone for this Control is implemented. This is due to a lack of FocusZone on UWP.